### ISSUE TITLE
Debugger: F10 and F11 keys not working, but F5 and F9 do

### ISSUE THREAD

**Issue by awik32** [joncampbell123/dosbox-x#4951]
### Describe the bug

After invoking the debugger with Alt+Pause, the debugger interface is displayed in the terminal.

However, the debugger does not respond to F10 or F11, but does respond to F9 and F5.

### Steps to reproduce the behaviour

Start DOSBox-X in a terminal emulator window.  In the graphical window, press Alt+Pause.  Try using F10 or F11.

### Expected behavior

The debugger is expected to respond to the F10 and F11 keys.

### What operating system(s) this bug have occurred on?

Linux Mint 21.3

### What version(s) of DOSBox-X have this bug?

dosbox-x-linux-x86_64-20240402155044.zip

### Used configuration

```ini
I have not created a configuration file.
```

### Additional information

I tried gnome-terminal, terminator, guake, and xterm.  The problem is the same with all of them.

### Have you checked that no similar bug report(s) exist?

- [X] I have searched and didn't find any similar bug report.

### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

**Comment by joncampbell123** [comment]
> False alarm! F10 and F11 do work, when I'm running this loop (from within DEBUG.EXE) in the VM: 0DAB:00000100 B055 mov al,55 0DAB:00000102 EBFC jmp short 00000100 ($-4) (up)
> 
> However, nothing happens if I'm at this instruction: F000:0000D186 CF iret
> 
> Why is that?

Is the previous instruction HLT?

HLT pauses CPU execution until an interrupt happens.

**Comment by awik32** [comment]
> > False alarm! F10 and F11 do work, when I'm running this loop (from within DEBUG.EXE) in the VM: 0DAB:00000100 B055 mov al,55 0DAB:00000102 EBFC jmp short 00000100 ($-4) (up)
> > However, nothing happens if I'm at this instruction: F000:0000D186 CF iret
> > Why is that?
> 
> Is the previous instruction HLT?

Indeed.

> HLT pauses CPU execution until an interrupt happens.

OK, I get it.

Thanks.

**Comment by joncampbell123** [comment]
Ah. As long as you're in the debugger, the HLT instruction never finishes because interrupt processing is never done while you're in the debugger.

**Comment by awik32** [comment]
You are right, single-stepping appears to be working normally when "core" is set to "normal".  The setting is "normal" after just starting DOSBox-X, but after I start my program, it gets changed automatically to "dynamic".  Can I prevent that from happening?

Another thing: "bpint *" puts a breakpoint on INT 0.  It would be more useful to set a breakpoint on *all* interrupts.

**Comment by joncampbell123** [comment]
strtoul("*") == 0 because * is not a number :)

**Comment by awik32** [comment]
I figured that was the reason.  An error message would be appropriate, or better yet, treat an "*" as a wildcard.

**Comment by awik32** [comment]
How do I set the NT flag?  "sr flags.nt=1" does not work.
